tests/Neo.UnitTests/SmartContract/Native/UT_StdLib.cs

```diff
@@ -209,6 +209,26 @@ public void StringSplit()
             Assert.AreEqual("b", arr[1].GetString());
         }
 
+        [TestMethod]
+        public void StringElementLength()
+        {
+            var snapshot = TestBlockchain.GetTestSnapshot();
+
+            using var script = new ScriptBuilder();
+            script.EmitDynamicCall(NativeContract.StdLib.Hash, "strLen", "🦆");
+            script.EmitDynamicCall(NativeContract.StdLib.Hash, "strLen", "ã");
+            script.EmitDynamicCall(NativeContract.StdLib.Hash, "strLen", "a");
+
```

Comment:
We have to have some tests for bad non-UTF8 things like `0xff` and their mix with regular characters.